js/views/modals/Settings/Settings.js

```diff
@@ -0,0 +1,87 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import SettingsGeneral from './SettingsGeneral';
+import SettingsPage from './SettingsPage';
+import $ from 'jquery';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      modelContentClass: 'modalContent clrP border clrBr',
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+
+    this.currentTabView = '';
```

Comment:
It's a bit odd that this will hold a view instance, but it's being initialized to an empty string. Does this really need to be initialized at all? `if (this.currentTabView) this.currentTabView.remove();` will work just the same if it's not initialized at all (since it will be undefined).